maxim/logger/mistral/utils.py

```
@@ -0,0 +1,151 @@
+import time
+from typing import Any, Dict, Iterable, List, Optional
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)